### PR TITLE
Pipeline for Ammersee (owner Niels Digemmanse)

### PR DESCRIPTION
**Pull request by LiamDBailey**

### CODE REVIEW

**Comment by LiamDBailey** [comment]
@ChloeRN I had to make some changes to other pipelines to avoid errors on windows. You can just focus on the changes to `format_AMM` and `test_AMM_pipeline` because these are the most relevant.

**Comment by LiamDBailey** [comment]
@StefanVriend added you to review. Maybe give it a try and see if it runs/passes the tests on your system. Feel free to leave other comments/reviews on the code if you feel like it 😄

File: R/format_AMM.R [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+                  HabitatType = "DEC") %>%
+    dplyr::select(.data$LocationID,
+                  .data$NestboxID,
+                  .data$LocationType,
+                  .data$PopID,
+                  .data$Latitude,
+                  .data$Longitude,
+                  .data$StartSeason,
+                  .data$EndSeason,
+                  .data$Habitat)
```
Comment (StefanVriend) [inline R/format_AMM.R:528]: After checking out origin/AMM_pipeline, I ran `run_pipelines(PopID = "AMM", save = FALSE)`, which resulted in an error related to these lines: `Error: Column 'Habitat' not found in '.data'`

**Review by StefanVriend — CHANGES_REQUESTED** [review]
Running `run_pipelines(PopID = "Amm")` resulted in the following error: `Error: Column 'Habitat' not found in '.data'`. See comment on "format_AMM.R".

**Comment by LiamDBailey** [comment]
New commits should fix those issues (didn't notice that I changed one of the column names 🤦). @StefanVriend could you try running it again? I think it should be fine now.

**Comment by StefanVriend** [comment]
Yes! I can confirm that the pipeline and the tests run flawlessly now. 
I cannot check whether the quality check runs, as I have not yet updated the quality check to v1.1.

**Comment by LiamDBailey** [comment]
No worries, I think we'll list it for now and worry about the quality check once it's updated to v1.1

**Pull request merged**